ltr_scripts/tf_ranking_libsvm.py

```diff
@@ -0,0 +1,466 @@
+# Copyright 2019 The TensorFlow Ranking Authors.
```

Comment:
Rather than include this file verbatim, would it be better to pull it in as a git submodule or something?